### PR TITLE
Update example tf

### PR DESCRIPTION
**Pull request by whaught**
Fixes https://github.com/google/exposure-notifications-verification-server/issues/1770

## Proposed Changes

* Update terraform example

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -186,29 +195,29 @@ Terraform module.
     inputs/parameters to the module definition. See the `variables.tf` file for
     the full list of configuration options.
 
-1.  (Optional if alerting is enabled): Manually create Google Cloud Monitoring
```
Comment (sethvargo) [inline terraform/README.md:189]: You can leave these all as "1". Markdown does the right thing.

**Review by sethvargo — APPROVED** [review]
/hold

File: terraform/README.md (hunk 1 above)
Comment (whaught) [reply terraform/README.md]: hmm must be my markdown vs plugin

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**